website/docs/d/connect_instance_storage_config.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+subcategory: "Connect"
+layout: "aws"
+page_title: "AWS: aws_connect_instance_storage_config"
+description: |-
+  Provides details about a specific Amazon Connect Instance Storage Config.
+---
+
+# Data Source: aws_connect_instance_storage_config
+
+Provides details about a specific Amazon Connect Instance Storage Config.
+
+## Example Usage
+
+```hcl
+data "aws_connect_instance_storage_config" "example" {
+  association_id = "1234567890123456789012345678901234567890123456789012345678901234"
+  instance_id    = "aaaaaaaa-bbbb-cccc-dddd-111111111111"
+  resource_type  = "CONTACT_TRACE_RECORDS"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `association_id` - (Required) The existing association identifier that uniquely identifies the resource type and storage config for the given instance ID.
+* `instance_id` - (Required) Reference to the hosting Amazon Connect Instance
```

Comment:
Recommend aligning with the SDK description: https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/connect#DescribeInstanceStorageConfigInput

```suggestion
* `instance_id` - (Required) The identifier of the Amazon Connect instance. 
```

Reply:
For the Connect set of resources, a consistent description for the `instance_id` is used to emphasise that the ID belongs to the hosting instance. This is because an account can potentially have multiple Connect instances.